js/src/modal.js

```diff
@@ -290,6 +294,31 @@ class Modal {
           this._element.focus()
         }
       })
+      .on(Event.KEYDOWN, (event) => {
+        if (event.isDefaultPrevented()) {
+          return
+        }
+        if (event.shiftKey && event.which === TAB_KEYCODE) {
+          const currentFocus = $(document.activeElement)
+          const tabbables = $(this._element).find(Selector.FOCUSABLE)
```

Comment:
Please use vanilla JS to query your elements, it's a pretty big selector here, so it'll be faster

Reply:
Fixed.

I made a simple benchmark: https://jsperf.com/finding-tabbable-elements/24
The fixed code runs twice faster!